static/custom.css

```diff
@@ -1,7 +1,17 @@
 body { background-color: var(--bs-dark) !important;
        color: var(--bs-light);
 }
-
 .row{
   margin-bottom: 5%;
+}
+
+.footer{
+  display: table;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+  position: absolute;
```

Comment:
@kwsorensen maybe this is what you were referring to in slack, but I think removing this `position` section fixes the text overlapping issues on mobile
```suggestion

```